tableauserverclient/server/endpoint/endpoint.py

```diff
@@ -78,16 +78,16 @@ def _make_request(
             self.parent_srv.http_options, auth_token, content, content_type, parameters
         )
 
-        logger.debug("request {}, url: {}".format(method, url))
+        logger.debug("request method {}, url: {}".format(method.__name__, url))
         if content:
             redacted = helpers.strings.redact_xml(content[:1000])
-            logger.debug("request content: {}".format(redacted))
+            # logger.debug("request content: {}".format(redacted))
```

Comment:
are the comments intentional?

Reply:
yes - they're too noisy to need most of the time, but worth adding specifically when debugging issues with server responses. The other option would be to implement a trace level for logging, but that was too much work.